packages/cli/src/lib/index.ts

```diff
@@ -17,18 +17,57 @@ import * as babel from '@babel/core'
 
 import c from './colors'
 
-export const asyncForEach = async (array, callback) => {
+export const asyncForEach = async (array: any[], callback: Function) => {
   for (let index = 0; index < array.length; index++) {
     await callback(array[index], index, array)
   }
 }
 
+export const readFile = (target: Parameters<typeof fs['readFileSync']>[0]) =>
```

Comment:
@peterp do you know where this comes from originally?

Reply:
I can't remember what the original motivation was, but we're going to abstract file-system access into something separate. I think it's OK for this to remain the way it is for now since it'll be removed at some point.